DataBiosphere/Saturn-UI
# [WM-2018] CBAS under Workflows Tab

### Jira Ticket: https://broadworkbench.atlassian.net/browse/WM-2018




## Summary of changes:


### What
- Skeleton pages and routes for each of the currently existing CBAS-UI pages
- SubmitWorkflow page complete as proof-of-concept (excepting Find a Workflow, covered by https://broadworkbench.atlassian.net/browse/WM-2023)

### Why
- Migrating CBAS UI into Terra UI

### Testing strategy
- Existing CBAS-UI test files copied over corresponding to the respective components/pages (Primarily `submission-common.test.js`)

In browser:
- Enter Azure workspace + Launch Cromwell app
- Enable Workflows feature flag in console: `configOverridesStore.set({ isAzureWorkflowsTabVisible: true })`
- Refresh and click on Workflows tab
- Page should be populated, Submission History button and cards will take you to page stubs

### Visual Aids
<img width="1512" alt="Screenshot 2023-06-28 at 4 58 33 PM" src="https://github.com/DataBiosphere/terra-ui/assets/67511512/2ca0da63-3242-4587-a0c8-c6e357db209a">
